app/views/layouts/application.html.erb

```diff
@@ -8,9 +8,23 @@
 
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= javascript_importmap_tags %>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous">
   </head>
 
   <body>
+    <% if user_signed_in? || current_page?(new_user_registration_path) || current_page?(new_user_session_path) %>
+      <% if not current_page?(root_path) %>
+        <%= render 'shared/nav-bar' %>
+      <% end %>
+    <% end %>
+    
     <%= yield %>
+
+    <%# <p class="notice"><%= notice %>
+    <%# </p>  %>
+    <%# <p class="alert"><%= alert %>
+    <%# </p>  %>
```

Comment:
- [ ] Same here, Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻

Reply:
@BenMukebo You are absolutely right, thank you very much for your comments. I have already applied your comments and applied your advice.

Reply:
This change has been done well